src/valuetypes.ts

```diff
@@ -435,15 +443,21 @@ export class ValueTypeEditor implements IValueTypeEditor {
     return this.impl.parse(def, data, accessor);
   }
 
-  guessOptions(def: ITypeDefinition, data: any[], accessor: (row: any) => any) {
+  async guessOptions(def: ITypeDefinition, data: any[], accessor: (row: any) => any) {
     def.type = this.id;
-    return this.impl.guessOptions(def, data, accessor);
+    const types = await this.impl.guessOptions(def, data, accessor);
+    this.desc.subType = types.subType;
```

Comment:
I'm confused the uploader shouldn't make any assumptions about the contained value editors. That's why you introduce the optgroup logic, to avoid that. In the future, one might add additional value types that are also return an optgroup and it shouldn't require a change in the uploader wizard logic